common/model-views.cpp

```diff
@@ -655,8 +660,32 @@ namespace rs2
         if (s->is<depth_sensor>())
         {
             auto colorizer = std::make_shared<processing_block_model>(
-                this, "Depth Visualization", depth_colorizer, error_message);
-            post_processing.push_back(colorizer);
+                this, "Depth Visualization", depth_colorizer, 
+                [=](rs2::frame f) { return depth_colorizer->colorize(f); }, error_message);
+            const_effects.push_back(colorizer);
```

Comment:
This causes a circular reference dependency (`subdevice_model` and `processing_block_model`). Make sure that destruction is safe